exposed-crypt/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%date{HH:mm:ss.SSS} %level %thread %logger{0}:%method:%line - %message %n</pattern>
+        </encoder>
+    </appender>
+    <appender name="ASYNC" class="ch.qos.logback.classic.AsyncAppender">
+        <appender-ref ref="CONSOLE"/>
+        <includeCallerData>true</includeCallerData>
+    </appender>
+
+    <root level="INFO">
+        <appender-ref ref="ASYNC" />
+    </root>
+    <logger name="Exposed" level="INFO" additivity="false">
+        <appender-ref ref="ASYNC"/>
+    </logger>
+</configuration>
```

Comment:
This file has been added to the test directory to set default log level to INFO. Otherwise, creating a `LogCaptor` in any unit test spams the console with debug logs from all tests in suite.